docs/.vuepress/public/presets/toml/no-empty-icons.toml

```
@@ -0,0 +1,150 @@
+[buf]
+format = '(with [$symbol($version )]($style))'
+
+[bun]
+format = '(via [$symbol($version )]($style))'
+
+[c]
+format = '(via [$symbol($version(-$name) )]($style))'
+
+[cmake]
+format = '(via [$symbol($version )]($style))'
+
+[cobol]
+format = '(via [$symbol($version )]($style))'
+
+[crystal]
+format = '(via [$symbol($version )]($style))'
+
+[daml]
+format = '(via [$symbol($version )]($style))'
+
+[dart]
+format = '(via [$symbol($version )]($style))'
+
+[deno]
+format = '(via [$symbol($version )]($style))'
+
+[docker_context]
+format = '(via [$symbol$context]($style) )'
+
+[dotnet]
+format = '(via [$symbol($version )(🎯 $tfm )]($style))'
+
+[elixir]
+format = '(via [$symbol($version \(OTP $otp_version\) )]($style))'
+
+[elm]
+format = '(via [$symbol($version )]($style))'
+
+[erlang]
+format = '(via [$symbol($version )]($style))'
+
+[git_branch]
+format = '(on [$symbol$branch(:$remote_branch)]($style) )'
+
+[git_commit]
+commit_hash_length = 7
+format = '([\($hash$tag\)]($style) )'
+
+[git_state]
+format = '(\([$state( $progress_current/$progress_total)]($style)\) )'
+
+[golang]
+format = '(via [$symbol($version )]($style))'
+
+[haskell]
+format = '(via [$symbol($version )]($style))'
+
+[helm]
+format = '(via [$symbol($version )]($style))'
+
+[hg_branch]
+format = '(on [$symbol$branch]($style) )'
+
+[java]
+format = '(via [$symbol($version )]($style))'
+
+[julia]
+format = '(via [$symbol($version )]($style))'
+
+[kotlin]
+format = '(via [$symbol($version )]($style))'
+
+[lua]
+format = '(via [$symbol($version )]($style))'
+
+[meson]
+format = '(via [$symbol$project]($style) )'
+
+[nim]
+format = '(via [$symbol($version )]($style))'
+
+[nix_shell]
+format = '(via [$symbol$state( \($name\))]($style) )'
+
+[nodejs]
+format = '(via [$symbol($version )]($style))'
+
+[ocaml]
+format = '(via [$symbol($version )(\($switch_indicator$switch_name\) )]($style))'
+
```

A new module was added since you created this PR.
```suggestion

[opa]
format = '(via [$symbo($version ]($style))"'
```

---

There's a few typos in this line, and it's missing a newline underneath.

They're fixed by https://github.com/starship/starship/pull/6031 and https://github.com/starship/starship/pull/6118 -- could someone take a look?